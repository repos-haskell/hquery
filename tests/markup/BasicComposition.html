<div class="person">
  <div class="name"></div>
  <div class="address"></div>
</div>
