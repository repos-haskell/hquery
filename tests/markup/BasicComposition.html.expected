<div class="person">
  <div class="name">Aaron Swartz</div>
  <div class="address">aaronsw@example.com</div>
</div>
